RHEL6_7/databases/postgresql/check

```diff
@@ -225,15 +225,15 @@ EOF
     append_to_solution <<EOF
 * Even if the postgresql-server is probably configured correctly, we are unable
   to say for 100% that the server is not used in some specific way on your
-  machine, so check that the
+  computer, so check that the
```

Comment:
Here you replace `machine` -> `computer` however on the other places, you leave the `machine`. I'm for leaving `machine` even here. To me, `computer` evokes something related to hardware. But I suspect that most customers are having virtual machines nowadays. Maybe just `system` would be even better, but that's probably something I would google first to see what's the commonly used term.